OpenRA.Mods.Common/Traits/Render/SupportPowerChargeBar.cs

```diff
@@ -36,13 +37,20 @@ public SupportPowerChargeBar(Actor self, SupportPowerChargeBarInfo info)
 
 		float ISelectionBar.GetValue()
 		{
-			if (!self.Owner.IsAlliedWith(self.World.RenderPlayer))
-				return 0;
-
 			var spm = self.Owner.PlayerActor.Trait<SupportPowerManager>();
```

Comment:
Out of curiosity, is there a reason why this can't be cached in ctor/on creation?